prerequisites-check.sh

```diff
@@ -15,11 +15,8 @@ check_command_present mvn
 check_command_present git
 check_command_present docker
 
-YQ_VERSION="$(yq --version | ${SED} 's/^.* //g')"
+YQ_VERSION="$(yq --version 2>&1 | ${SED} 's/^.* //g')"
 # After version 3.3.1, yq --version sends the string to STDERR instead of STDOUT
```

Comment:
You should probably remove the comment which was related to the code below which you removed.

Reply:
This comment is still relevant to the changed line, is it worth moving it to the line above or is the code now sufficiently self-explanatory that the comment can just be removed?

Reply:
Yeah ... I'm fine if you move it ... it just makes no sense below.

Reply:
Comment moved